common/data_refinery_common/management/commands/programs.tsv

```diff
@@ -0,0 +1,7 @@
+# progam_name	program_version	command
+salmon	0.9.1	salmon --no-version-check quant -l A --biasSpeedSamp 5 -i <index_dir> -1 <input_file_1> -2 <input_file_2> -p 20 -o <output_dir> --seqBias --gcBias --dumpEq --writeUnmappedNames
+salmon	0.9.1	salmon --no-version-check quant -l A --biasSpeedSamp 5 -i <index_dir> -r <file> -p 20 -o <output_dir> --seqBias --gcBias --dumpEq --writeUnmappedNames
```

Comment:
No GC-bias correction on single-end reads, right? https://github.com/AlexsLemonade/refinebio/blob/caa2fd630de8d9e6e8bc1b2dfbedf24455e36baa/workers/data_refinery_workers/processors/salmon.py#L314

Reply:
@jaclyn-taroni You are definitely right> I made a mistake during copy/paste. Thanks!